app/controllers/api/v1/merchants_controller.rb

```diff
@@ -23,8 +23,12 @@ def show
     end
 
     def create
-        new_merchant = Merchant.create(merchant_params)
-        render json: MerchantSerializer.new(new_merchant), status: 201
+        merchant = Merchant.create(merchant_params)
+        
+        if merchant.save
+        render json: MerchantSerializer.new(merchant), status: 201
+        else
+            render json: { errors: merchant.errors.full_messages }, status: :bad_request        end
```

Comment:
good implementation of error handle, we will have the error serializer for quicker implementation of this moving forward, but learning through experience is valuable so Im glad we're doing this manually